scripts/build.sh

```diff
@@ -31,16 +31,11 @@ fi
 set -ex -o pipefail
 
 # Clean up from last build
-rm -rf dist
+yarn bazel clean
```

Comment:
Nit (just a statement): Generally I dislike cleaning before releasing as it avoids the benefits of cache and potentially is a symptom of bad build rules. I know that for stamping with `pkg_npm` this is historically needed though since the version is set as a volatile key instead of stable key (so that's a bug- had an issue for that somewhere).

(In Components we don't clean- but we just wipe the `pkg_npm` target to make sure it's re-stamped)

Reply:
Yea, I had added that because of the stamping that might result in multiple `vsix` files being in the `bazel-bin` (ones from previous builds and one from the current build since the `vsix` name included the version). I didn't want there to be any accident with copying/publishing the wrong version. 

Now that the build has removed the version from the `vsix` name, I think this can more safely be removed.